syntaxes/vscode-scl.tmLanguage.json

```diff
@@ -312,7 +331,7 @@
 			"patterns": [
 				{
 					"name": "constant.character.escape.scl",
-					"match": "\\\\."
+					"match": "'[^'\\$]*(?:\\$[\\s\\S][^'\\$]*)*'"
```

Comment:
This is incorrect and does not catch all escaped characters. 

Try this simple text i created as a test `' some text \t with some excaped \c characters in \' \"\i it' notAString`

Do you have an example of when `\\\\.` does not work?

Reply:
The thing is, Siemens doesn't use `\` to escape.. They use `$`.. So `\\\\.` won't work. See https://github.com/Gunders89/vscode-scl/pull/17#pullrequestreview-2215730160

Reply:
ah. Is this true for Step7 as well or is this just for TIA?

Reply:
I think it applies for Step7 aswell, no personal experience with it tho.
https://support.industry.siemens.com/cs/mdm/109798671?c=141583806731&lc=en-AF
https://support.industry.siemens.com/cs/document/12228598/entering-signs-in-string-variables

Reply:
I see. An oversight on my part then. Maybe both needs to be handled. I usually use TwinCAT these days and don't have licenses to test on siemens any more.

Reply:
Sure could try and handle both, maybe that's something you could look into?

Reply:
yup. I can fix that. Gj

Reply:
I solved this like this:
```
"strings": {
			"name": "string.quoted.single.scl",
			"begin": "'",
			"end": "'",
			"patterns": [
				{
					"name": "constant.character.escape.scl",
					"match": "\\\\."
				},
				{
					"name": "constant.character.escape.scl",
					"match": "\\$."
				}
			]
		}
```

Create an issue if it's still providing the wrong result for you